### ISSUE TITLE
Android BOINC 7.16.16 stays in memory all the time

### ISSUE THREAD

**Issue by Ageless93** [BOINC/boinc#4189]
It's not possible to get BOINC out of memory, it's become an always on application. I can open the message about it being suspended in my phone's notification bar but I cannot swipe it left or right out of memory. Pressing square and choosing the garbage bin to remove all active apps from memory will leave BOINC running and using battery. 

https://ibb.co/5G03SjN

**Comment by Ageless93** [comment]
It gets better. Having unplugged my phone, after half an hour I could swipe away the notification on the notification bar that BOINC was suspended. I had already cleared memory, so normally for other apps this means they are really really not running. Not so for BOINC. 

https://user-images.githubusercontent.com/10869587/108099574-1928d600-7085-11eb-9fa7-80763508d89b.jpg
6.95% of BOINC using my battery for an app that is suspended... ouch.

**Comment by Ageless93** [comment]
Ran BOINC overnight while my phone was being charged. 
https://i.ibb.co/y8xk276/Battery-usage-30.jpg shows it used 30% of the battery during that charging time. It's also running while I manually suspended BOINC and it's not on the charger. I then force stop BOINC and wait half an hour. 
https://i.ibb.co/TbxtvC8/Battery-usage-1.jpg is what I would count as normal usage of the battery (1%), with a forced stopped BOINC.
